src/tools/pontoon/translation_workspace.md

```diff
@@ -192,6 +193,21 @@ Icons to the right indicate the [status](#translation-status) of each translatio
 
 In the screenshot above, the first item is the approved translation (green checkmark), while the other two are rejected suggestions.
 
+#### Status banners
```

Comment:
The word status is already used in the context of [translations](https://github.com/mozilla-l10n/localizer-documentation/blob/b4e3a6452b4b9754640c93d4ecc481fb4141fcdd/src/tools/pontoon/translation_workspace.md#translation-status), so we should make it explicit this is about users. Perhaps we can even drop the word status?

```suggestion
#### User banners
```

Reply:
If we do drop the word status, what would you suggest for line 179?

Reply:
Added a suggestion below.